AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStore/src/DefaultKeyStorageInterface.dfy

```diff
@@ -828,11 +833,10 @@ module DefaultKeyStorageInterface {
         Types.KeyStorageException(
           message:="DDB returned no items from table: " + ddbTableName));
 
-      // SDKs/Smithy-Dafny/Custom Implementations of Storage MAY respond with None or an Empty Map.
-      // .NET returns an empty map, Java returns None.
-      var thereAreNoUnprocessedKeys: bool := ddbResponse.UnprocessedKeys.None? || |ddbResponse.UnprocessedKeys.value| == 0;
+        // SDKs/Smithy-Dafny/Custom Implementations of Storage MAY respond with None or an Empty Map.
+        // .NET returns an empty map, Java returns None.
       :- Need(
-        thereAreNoUnprocessedKeys,
+        ddbResponse.UnprocessedKeys.None? || |ddbResponse.UnprocessedKeys.value| == 0,
         Types.KeyStorageException(
           message:=
             "DDB returned UnprocessedKeys for a BatchGetItem request of 3 items"
```

Comment:
I thought you said no more BatchGetItem and instead a TransactGet? does this message need updating?

Reply:
Yes! I will try and find this and correct it!